### ISSUE TITLE
when connecting on IOS it stops 3rd party music apps

### ISSUE THREAD

**Issue by NirGol** [flutter-webrtc/flutter-webrtc#1152]
only happens on ios, when a 3rd party music app running, apple music, spotify, and webrtc connect is shut down the other app audio, is there a way to prevent it? on android it works

**Comment by cloudwebrtc** [comment]
Do you mean after the webrtc connection is established?

**Comment by NirGol** [comment]
exactly

**Comment by cloudwebrtc** [comment]
This should be normal, AudioUnit is used on iOS, VoiceChat Mode needs to be enabled, and HW echo cancellation needs to be enabled at this time, I think iOS must turn off the music or another playback to ensure the quality of voice calls

**Comment by NirGol** [comment]
yes it does resumes, the issue is I use it for streaming, not calls

**Comment by cloudwebrtc** [comment]
If you turn on the microphone, webrtc will automatically enable voice chat mode, which seems to be an internal mechanism of WebRTC.

**Comment by fullflash** [comment]
it happens for android also 13 Samsung device.
